src/SDKs/Batch/Management/Management.Batch.Tests/Management.Batch.Tests.csproj

```diff
@@ -9,12 +9,16 @@
   </PropertyGroup>
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
+    <Version>1.0.0.0</Version>
+    <Company>MSIT</Company>
+    <Copyright>Copyright © MSIT 2013</Copyright>
+    <SignAssembly>false</SignAssembly>
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="[1.1.3-preview,2.0.0)" />
```

Comment:
@xingwu1 upgrade ResourceManager to 1.6.0 (minimum version) very soon, all test projects will get implicit reference to 1.6.0

Reply:
Fixed.